eureka_ml_insights/configs/aime.py

```diff
@@ -0,0 +1,161 @@
+import os
+from typing import Any, Optional
+
+from eureka_ml_insights.core import (
+    DataProcessing,
+    Inference,
+    PromptProcessing,
+)
+from eureka_ml_insights.core.eval_reporting import EvalReporting
+from eureka_ml_insights.data_utils import (
+    AddColumn,
+    ColumnRename,
+    DataReader,
+    HFDataReader,
+    MMDataLoader,
+    SequenceTransform,
+    SamplerTransform,
+    
+)
+
+
+from eureka_ml_insights.data_utils.aime_utils import (
+    AIMEExtractAnswer
+)
+
+
+
+from eureka_ml_insights.data_utils.data import (
+    DataLoader,
+    DataReader,
+    HFDataReader,
+)
+from eureka_ml_insights.data_utils.transform import RunPythonTransform
+from eureka_ml_insights.metrics.aime_metrics import AIME_ExactMatch
+
+from eureka_ml_insights.metrics.reports import (
+    AverageAggregator,
+    TwoColumnSumAverageAggregator,
+)
+
+from .config import (
+    AggregatorConfig,
+    DataProcessingConfig,
+    DataSetConfig,
+    EvalReportingConfig,
+    InferenceConfig,
+    MetricConfig,
+    ModelConfig,
+    PipelineConfig,
+    PromptProcessingConfig,
+)
+from .experiment_config import ExperimentConfig
+
+
+class AIME_PIPELINE(ExperimentConfig):

```

Comment:
Could you please add a pipeline test like other benchmarks have under tests/?

Reply:
Yes. A pipeline test was added